src/common/commonutils/DeviceInfoUtils.c

```diff
@@ -141,6 +136,7 @@ char* GetOsName(void* log)
             RemovePrefixBlanks(textResult);
             RemoveTrailingBlanks(textResult);
             RemovePrefixUpTo(textResult, '=');
+            RemovePrefix(textResult, '=');

```

Comment:
See my comment under unifying RemovePrefixUpTo with RemovePrefixUpToString (this is resolved now).

Reply:
This is needed here now, once the RemovePrefixUpTo was changed to return the token.

The alternative would have been to add another function, to RemovePrefixAndSkipOverToken or similar, just prefer this variant as it's more explicit of what happens.